src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -121,16 +121,30 @@
 				<ul class="h-100" :class="{'scroller': isSearching}">
 					<!-- Conversations List -->
 					<template v-if="!isSearching">
-						<li class="h-100">
+						<NcAppNavigationCaption v-if="isFiltered && filteredConversationsList.length > 0" :name="getFilterCaption" />
```

Comment:
It also creates a scrollbar, because virtual list is rendered.
Check if changing `height: 100%` with `max-height: 100%` in class 'h-100' won't break anything, and I'd suggest to proceed with it

Reply:
It breaks the smooth scrolling when conversation is not fully visible in the viewport ( check `ConversationsListVirtual` )